styles/modules.css

```diff
@@ -42,3 +43,20 @@
 #aboutInfo{
   display: none;
 }
+.burger{
+  float:right;
+}
+.bar{
+  background-color: white;
+  margin: 6px;
+  height: 5px;
+  width: 35px;
+}
+@media screen and (min-width: 700px){
```

Comment:
You need to add more here to make your website all size screen friendly. Right now it is just mobile friendly.